crates/biome_js_formatter/src/js/module/export_named_from_clause.rs

```diff
@@ -21,6 +21,7 @@ impl FormatNodeRule<JsExportNamedFromClause> for FormatJsExportNamedFromClause {
             assertion,
             semicolon_token,
         } = node.as_fields();
+        let should_insert_space_around_brackets = f.options().bracket_spacing().value();
```

Comment:
Code style preference: I put this access to the configuration as a local variable everywhere it gets used, but I could also just pass it in directly to the function calls instead. Not sure which is preferred.

Reply:
We don't have a preference :)